client/src/font/icons-reference.html

```diff
@@ -1011,6 +1007,14 @@ <h2>CSS mapping</h2>
           <div class="icon font-icon-attention-1"></div>
           <input type="text" readonly="readonly" value="attention-1">
         </li>
+        <li>
+          <div class="icon font-icon-silverstripe"></div>
```

Comment:
I've kept the old Siverstripe icon in there, in case we decide we have a use case for showing the company logo.